docs/sources/variables/add-template-variables/index.md

```diff
@@ -0,0 +1,510 @@
+| Data source       | Quickly change the data source for an entire dashboard. [Add a data source variable]({{< relref "#add-a-data-source-variable" >}}).                                                                |
+| Interval          | Interval variables represent time spans. [Add an interval variable]({{< relref "#add-an-interval-variable" >}}).                                                                                   |
+| Ad hoc filters    | Key/value filters that are automatically added to all metric queries for a data source (InfluxDB, Prometheus, and Elasticsearch only). [Add ad hoc filters]({{< relref "#add-ad-hoc-filters" >}}). |
+| Global variables  | Built-in variables that can be used in expressions in the query editor. Refer to [Global variables]({{< relref "#global-variables" >}}).                                                           |
+| Chained variables | Variable queries can contain other variables. Refer to [Chained variables]({{< relref "#chained-variables" >}}).                                                                                   |
+
+## Add a query variable
+
+Query variables enable you to write a data source query that can return a list of metric names, tag values, or keys. For example, a query variable might return a list of server names, sensor IDs, or data centers. The variable values change as they dynamically fetch options with a data source query.
+
+Query variables are generally only supported for strings. If your query returns numbers or any other data type, you might need to convert them to strings in order to use them as variables. For the Azure data source, for example, you can use the [tostring](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/tostringfunction) function for this purpose.
+
+Query expressions can contain references to other variables and in effect create linked variables. Grafana detects this and automatically refreshes a variable when one of its linked variables change.
+
+### Query expressions
+
+Query expressions are different for each data source. For more information, refer to the documentation for your [data source]({{< relref "../../datasources/" >}}).
+
+### Enter General options
+
+1. Navigate to the dashboard you want to make a variable for and click the **Dashboard settings** (gear) icon at the top of the page.
+1. On the **Variables** tab, click **New**.
+1. Enter a **Name** for the variable.
+1. In the **Type** list, select **Query**.
+1. (Optional) In **Label**, enter the display name of the variable dropdown.
+
+   If you don't enter a display name, then the dropdown label is the variable name.
+
+1. Choose a **Hide** option:
+   - **No selection (blank):** The variable dropdown displays the variable **Name** or **Label** value. This is the default.
+   - **Label:** The variable dropdown only displays the selected variable value and a down arrow.
+   - **Variable:** No variable dropdown is displayed on the dashboard.
```

Comment:
These general options looks duplicated a bunch in this file. For this first section it also looks a bit strange interms of spacing (lots of empty space between bullets) 

![Screenshot from 2022-08-31 15-57-32](https://user-images.githubusercontent.com/10999/187696555-df18e65d-d79a-4c6a-8898-7f7733c55050.png)

Reply:
stepped out the general options as its own section and then refer back to it in step 1 of every task. Was able to remove the duplicate content.